src/execution_plan/ops/op_create.c

```diff
@@ -48,11 +46,7 @@ static void _CreateNodes(OpCreate *op, Record r) {
 		QGNode *n = op->pending.nodes_to_create[i].node;
 
 		/* Create a new node. */
-		Node newNode = {
-			.entity = NULL,
-			.label = n->label,
-			.labelID = n->labelID
-		};
+		Node newNode = Node_NewLabeledNode(n->label, n->labelID);
```

Comment:
What's the defaults for `label` and `labelID` ?

Reply:
QueryGraph nodes are initialized with `NULL` and `GRAPH_NO_LABEL`, then both are updated in the QueryGraph itself if the query specifies a label.